wdl-analysis/src/task/v1.rs

```diff
@@ -0,0 +1 @@
+//! Task graph representation for a V1 AST.
```

Comment:
Not sure about this as well.

Reply:
This is also a dead code from very early on which was accidentally committed; I'll remove.